product-cli/cmd/cluster/agent/destroy.go

```diff
@@ -0,0 +1,28 @@
+package agent
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/hypershift/cmd/cluster/agent"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	"github.com/openshift/hypershift/cmd/log"
+)
+
+func NewDestroyCommand(opts *core.DestroyOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "agent",
+		Short:        "Destroys a HostedCluster and its associated infrastructure on Agent.",
```

Comment:
I don't see we using period of the sentences in "Short" here https://github.com/openshift/hypershift/pull/2755/files#diff-a0b6865b7f8d3556ea1be8721258c0d1ff94f8c4b773b8ceb1947feef238e974R13
can we be consistent?